<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/logo_icone.ico</normaloff>image/logo_icone.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.323455, y1:0.722, x2:0.807045, y2:0.227, stop:0.119318 rgba(255, 0, 0, 255), stop:1 rgba(0, 0, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/LogoPrincipal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EntryUser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="EntryPass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="Show">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>70</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Next -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Register">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EntryEmail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Email">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EntryCity">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="City">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>505</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string>City:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Voice">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="text">
     <string>Use with voice command</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>&lt;- Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelRegister">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: none;</string>
    </property>
    <property name="text">
     <string>Register System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Forgot">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>395</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: none;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Forgot Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>image/fundo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>Logo</zorder>
   <zorder>Username</zorder>
   <zorder>Password</zorder>
   <zorder>EntryUser</zorder>
   <zorder>EntryPass</zorder>
   <zorder>Show</zorder>
   <zorder>Next</zorder>
   <zorder>Register</zorder>
   <zorder>EntryEmail</zorder>
   <zorder>Email</zorder>
   <zorder>EntryCity</zorder>
   <zorder>City</zorder>
   <zorder>Voice</zorder>
   <zorder>Back</zorder>
   <zorder>LabelRegister</zorder>
   <zorder>Forgot</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
